Jenkinsfile

```diff
@@ -12,11 +12,16 @@ pipeline {
 		    mkdir tmp
                     mv !(tmp) tmp
 
-		    # clone and update submodules here
-                    git clone https://github.com/couchbaselabs/couchbase-lite-ios-ee.git --branch $CHANGE_TARGET
-		    # submodule update inside lite-ios
+		    # Sometimes the PR depends on a PR in the EE repo as well. This needs to be convention based, so if there is a branch with the name PR-###
+                    # (with the GH PR number) in the EE repo then use that, otherwise use the name of the target branch (master, release/XXX etc) 
+		    # clone the EE-repo
+                    git clone git@github.com:couchbaselabs/couchbase-lite-ios-ee.git --branch $BRANCH_NAME || \
+		      git clone git@github.com:couchbaselabs/couchbase-lite-ios-ee.git --branch $CHANGE_TARGET
+
+		    # clone the core-EE
 		    pushd couchbase-lite-ios-ee
-                    git submodule update --init --recursive
+		    git clone git@github.com:couchbase/couchbase-lite-core-EE.git --branch $BRANCH_NAME || \
```

Comment:
This is hard to achieve as CBL-iOS doesn't control lite-core-EE's branch. If we want to update lite-core-EE, can we update the commit to the ios-ee's branch?

Reply:
not sure, how we can achieve that, even possible?. Since we need some intelligence in choosing it, other than some manual intervention. Here `lite-ios-EE` and `lite-core-EE` uses the same logic. If there is a branch called, `pr-xx`, it will pick up, else go with master.